libs/openFrameworks/sound/ofRtAudioSoundStream.h

```diff
@@ -9,6 +9,18 @@
 class RtAudio;
 typedef unsigned int RtAudioStreamStatus;
 
+enum ofRtAPI {
```

Comment:
Also, I'm wondering if it's necessary to duplicate an enum already in the underlying code, especially since it's not part of the base class.  For people needing this functionality, couldn't they just call `RtAudio:: LINUX_ALSA` directly?  At the very least, if an oF enum is going to ghost an existing one, it would probably be specified like:

```c++
enum ofRtAPI {
OF_RT_API_UNSPECIFIED = RtAudio:: UNSPECIFIED,
OF_LINUX_ALSA = RtAudio:: LINUX_ALSA,
...

```